docs/dev_guide/contents/example_gallery.rst

```diff
@@ -37,7 +37,7 @@ Contribution Guidelines
 
   * Always create a figure instance using ``plt.figure()`` prior to creating a plot.
 
-  * Only create an axes instance if it is explicitly needed later in the example (e.g. when overplotting a coordinate on a map using ``ax.plot_coord``).
+  * Always create an Axes instance, and where possible use this to modify the plot instead of using the ``pyplot`` interface (for example use ``ax.set_label()``` instead of ``plt.xlabel()``).
```

Comment:
```suggestion
  * Always create an Axes instance, and where possible use this to modify the plot instead of using the ``pyplot`` interface (for example use ``ax.set_xlabel()``` instead of ``plt.xlabel()``).
```